src/CSPropAbsorbingBC.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+*	Copyright (C) 2023-2025 Gadi Lahav (gadi@rfwithcare.com)
+*
+*	This program is free software: you can redistribute it and/or modify
+*	it under the terms of the GNU Lesser General Public License as published
+*	by the Free Software Foundation, either version 3 of the License, or
+*	(at your option) any later version.
+*
+*	This program is distributed in the hope that it will be useful,
+*	but WITHOUT ANY WARRANTY; without even the implied warranty of
+*	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*	GNU Lesser General Public License for more details.
+*
+*	You should have received a copy of the GNU Lesser General Public License
+*	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "tinyxml.h"
+
+#include "CSPropAbsorbingBC.h"
+
+#define _C0_ 299792458.0
+
+CSPropAbsorbingBC::CSPropAbsorbingBC(ParameterSet* paraSet) : CSProperties(paraSet) {Type=(CSProperties::PropertyType)(ABSORBING_BC); Init();}
+CSPropAbsorbingBC::CSPropAbsorbingBC(CSProperties* prop) : CSProperties(prop) {Type=(CSProperties::PropertyType)(ABSORBING_BC); Init();}
```

Comment:
I have changes the copy constructors in all properties. Maybe you have a look how it is handled now?